README.md

```diff
@@ -21,7 +21,35 @@ $ pip install -r requirements.txt
 
 * After tunnelling with PuTTY, you will be asked for login credentials. Adding your public key for SSH access will make your login process slightly easier.
 * You may wish to use a desktop client such as [HeidiSQL](https://www.heidisql.com/) for convenience.
-  * Contact the owners Fantasy Anime League for an individual db user account to be created for you.
+  * Contact the owners of Fantasy Anime League for an individual db user account to be created for you.
+
+## Order to run scripts
+
+### Preseason (before registration)
+```shell
+$ python run_fal.py --collect-series
+$ python run_fal.py --ptw-counter
+```
+
+### Preseason (after registration)
```

Comment:
Thanks for this. Might need to include running some files in the registration repo at this point though (e.g. Converter_code.py) I'm hoping after we get parity that we can streamline that process and make it more direct